stdlib/importlib/metadata/__init__.pyi

```diff
@@ -35,6 +35,23 @@ if sys.version_info >= (3, 8):
             dist: ClassVar[Distribution | None]
             def matches(self, **params: Any) -> bool: ...  # undocumented
 
+    if sys.version_info >= (3, 10):
+        class EntryPoints(list[EntryPoint]):  # use as list is deprecated since 3.10
+            def select(self, **params: Any) -> EntryPoints: ...
+            @property
+            def names(self) -> set[str]: ...
+            @property
+            def groups(self) -> set[str]: ...
+
+        class SelectableGroups(dict[str, EntryPoints]):  # use as list is deprecated since 3.10
+            @classmethod
+            def load(cls, eps: SelectableGroups) -> SelectableGroups: ...
```

Comment:
I think this is right:
```suggestion
            def load(cls: type[Self], eps: Iterable[EntryPoint]) -> Self: ...
```
Where `Self` is from `_typeshed`.

- It returns `cls()`, so it should return `Self` (this makes a difference for a subclass of `SelectableGroups`)
- The arguments get passed to the `EntryPoints` constructor, which wants an iterable of `EntryPoint` instances.

Reply:
Note that this isn't done in the rest of the stub file.

Reply:
Done.